Gemfile

```diff
@@ -43,14 +43,18 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 gem "bootsnap", require: false
```

Comment:
Install rubocop and create a new PR with fixing the offenses. Please ask Zain to guide you.

Reply:
okay sure